youtube_dl/extractor/bbc.py

```diff
@@ -959,6 +959,32 @@ def _real_extract(self, url):
              r'videoId\s*:\s*["\'](%s)["\']' % self._ID_REGEX],
             webpage, 'vpid', default=None)
 
+        # bbc reel (e.g. https://www.bbc.com/reel/video/p07c6sb6/how-positive-thinking-is-harming-your-happiness)
+        if not programme_id:
+            initial_data = self._search_regex(
```

Comment:
use `_html_search_regex` and get rid of `unescapeHTML`.

Reply:
As below.